config/environments/development.rb

```diff
@@ -40,14 +53,20 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
Did you intend to uncomment this?

Reply:
Good spot - the Rails update automatically uncommented it. It seemed a harmless change (dev environment only) and CI continued to pass, so I didn't think much of it. Have now tried a `govuk-docker-up` locally though and sure enough, this caused an issue as we don't have a `listen` gem installed 🤦 
Will now re-comment this line.